src/app/games/gos/tri-go/tri-go.component.ts

```diff
@@ -0,0 +1,159 @@
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { GoMove } from 'src/app/games/gos/GoMove';
+import { TriGoConfig, TriGoRules } from './TriGoRules';
+import { GoState } from 'src/app/games/gos/GoState';
+import { GoPiece } from '../GoPiece';
+import { Coord } from 'src/app/jscaip/Coord';
+import { MGPOptional, MGPValidation, Utils } from '@everyboard/lib';
+import { GroupDatas } from 'src/app/jscaip/BoardDatas';
+import { MessageDisplayer } from 'src/app/services/MessageDisplayer';
+import { MCTS } from 'src/app/jscaip/AI/MCTS';
+import { GoMoveGenerator } from '../go/GoMoveGenerator';
+import { Debug } from 'src/app/utils/Debug';
+import { PlayerNumberMap } from 'src/app/jscaip/PlayerMap';
+import { GoPhase } from '../GoPhase';
+import { TriangularGameComponent } from 'src/app/components/game-components/game-component/TriangularGameComponent';
+import { GoLegalityInformation } from '../AbstractGoRules';
+import { ViewBox } from 'src/app/components/game-components/GameComponentUtils';
+import { TriangularCheckerBoard } from 'src/app/jscaip/state/TriangularCheckerBoard';
+import { TriGoMinimax } from './TriGoMinimax';
+
+@Component({
+    selector: 'app-tri-go',
+    templateUrl: './tri-go.component.html',
+    styleUrls: ['../../../components/game-components/game-component/game-component.scss'],
+})
+@Debug.log
+export class TriGoComponent extends TriangularGameComponent<TriGoRules,
```

Comment:
ça se ferait d'extend GoComponent pour éviter plus de duplication ?

Reply:
j'ai procédé comme ça au début puis in fine ça share plus de code avec du sahara ou triangular stuff clairement nope